pom.xml

```diff
@@ -362,6 +363,24 @@
       <version>1.12.14</version>
       <scope>test</scope>
     </dependency>
+    <!--
+      This is just for dependency updates.
+
+      TODO: Delete this dependency and figure out how to automatically update
+      this dependency some other way.
+    -->
+    <dependency>
+      <groupId>org.jenkins-ci.main</groupId>
+      <artifactId>jenkins-core</artifactId>
+      <version>${jenkins.version}</version>
+      <scope>test</scope>
+      <exclusions>
+        <exclusion>
+          <groupId>*</groupId>
+          <artifactId>*</artifactId>
+        </exclusion>
+      </exclusions>
+    </dependency>
```

Comment:
Copying the trick I did in `plugin-compat-tester` to define an unused test-scoped dependency to get Dependabot to automatically update `jenkins.version` (actually used further below by the `copy` mojo, which Dependabot can't update). I have confirmed this trick works. I am not interested in learning how to use any other dependency updating tools. I trust that those who are enthusiastic about other tools can find a better way to do this after this PR lands (or stick with this solution, which is "good enough", if they are not interested).